synchronization/src/main/java/de/cyface/synchronization/NetworkCallback.java

```diff
@@ -19,8 +19,6 @@
  * be enabled depending on the {@link android.net.NetworkCapabilities#NET_CAPABILITY_NOT_METERED} capabilities of the
  * newly connected network.
  *
- * FIXME: We need to test this on API >= 21 devices !!
```

Comment:
There is an open task for the bug that sync happens without wifi so this can be removed